mne/forward/tests/test_make_forward.py

```diff
@@ -203,6 +203,13 @@ def test_make_forward_solution_kit():
                                    subjects_dir=subjects_dir)
     _compare_forwards(fwd, fwd_py, 274, n_src)
 
+    temp_dir = _TempDir()
+    fname_temp = op.join(temp_dir, 'test-ctf-fwd.fif')
+    write_forward_solution(fname_temp, fwd_py)
+    fwd_py2 = read_forward_solution(fname_temp)
+    _compare_forwards(fwd_py, fwd_py2, 274, n_src)
+    fwd_py.__repr__()  # __repr__ also calls _check_consistency
```

Comment:
For future reference `repr(...)` is a slightly cleaner way to do this (but don't need to change here)

Reply:
I would change it